docs/openapi/openapi.yaml

```diff
@@ -219,16 +219,69 @@ components:
     InitialLpa:
       type: object
       required:
+        - type
```

Comment:
Could we come up with something more specific than "type"? It's something that often trips us up, since we have lots of other things called "type" like case types (EPA vs LPA vs Digital LPA) and application types (Paper vs Online). Also programming language types, the "type" field in this YAML spec, and it's often a reserved word so ends up having to be called something else anyway. I'll ask in the cross-team channel.

Reply:
I did almost put `lpaType`, but then changed because type on lpa means lpa type ... but will happily change it back

Reply:
I think it was also discussed somewhere to expand pfa/hw to PropertyAndAffairs/PersonalWelfare, which I could do if you preferred?

Reply:
Yeah both are useful for being more explicit, especially for any downstream use by say analytical platform users where acronyms might confuse or new users our side.